.eslintrc

```diff
@@ -16,17 +16,21 @@
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
         "plugin:prettier/recommended",
-        "prettier/@typescript-eslint"
+        "prettier/@typescript-eslint",
+        "plugin:you-dont-need-lodash-underscore/compatible"
     ],
     "rules": {
-        "prettier/prettier": ["error", {
-            "singleQuote": true,
-            "printWidth": 100,
-            "tabWidth": 2,
-            "semi": true,
-            "bracketSpacing": false
-        }],
+        "prettier/prettier": [
+            "error",
+            {
+                "singleQuote": true,
+                "printWidth": 100,
+                "tabWidth": 2,
+                "semi": true
+            }
+        ],
         "@typescript-eslint/no-this-alias": "warn",
-        "@typescript-eslint/interface-name-prefix": "warn"
+        "@typescript-eslint/interface-name-prefix": "warn",
+        "@typescript-eslint/explicit-function-return-type": "off"
     }
-}
+}
```

Comment:
Seems you dont like training New lines :wink:

Reply:
I'm just lazy ;) Wonder why my editor doesn't take care of it though... must be a setting I can enable in vscode. 🤔

Reply:
The laziest things would be to let the file as is ^^
Maybe thats your vscode configuration that make this? 🤔